tests/foreman/ui/test_host.py

```diff
@@ -1701,3 +1704,194 @@ def test_positive_delete_libvirt(
             'setting "Destroy associated VM on host delete".'.format(name)
         ) == message
         assert not session.host.search(name)
+
+
+@pytest.fixture
+def gce_client(download_cert):
+    return GoogleCloudSystem(
+        project=settings.gce.project_id,
+        zone=settings.gce.zone,
+        file_path=download_cert,
+        file_type='json')
+
+
+@pytest.fixture
+def class_gce_template(gce_client):
```

Comment:
Actually, this aligns with existing fixtures in this file which starts with ```module_``` which suggests that it can be used at module level.

Here the class says that it just aligns with particular CR. I will remove class itself from all and will start with ```gce_```.